src/PIL/ImageGrab.py

```diff
@@ -94,7 +101,7 @@ def grab(bbox=None, include_layered_windows=False, all_screens=False, xdisplay=N
         return im
 
 
-def grabclipboard():
+def grabclipboard() -> Image.Image | list[str] | None:
```

Comment:
This could be `ImageFile.ImageFile`, rather than `Image.Image`

Reply:
I'm not opposed to changing this to `ImageFile`, but would that unnecessarily lock us in to a given return type?
Or can we change return types without a major version bump and deprecation period?

Reply:
I hope that we can change return types without deprecation, but I can appreciate that we could be a bit forwards-compatible here. There's also no reason for the user to expect `ImageFile` instead of an `Image`. So I think either form is fine.